build.gradle

```diff
@@ -6,7 +6,7 @@ buildscript {
     dependencies {
         classpath 'com.android.tools.build:gradle:3.0.1'
         classpath 'com.github.dcendents:android-maven-gradle-plugin:2.0'
-
+        classpath 'net.openid:appauth:0.7.0'
```

Comment:
I don't think this should be at the top level build.gradle file. This is only required by the auth module, so I say move it to the `auth/build.gradle` file and it should be an implementation dependency.

Reply:
Just +1ing wei's comment.  Additionally this block is the dependencies for building (think devDependencies from node.js) as opposed to dependencies for the code.